Makefile

```diff
@@ -1,2 +1,5 @@
-prom:
-	docker run -p 9090:9090 --network=host -v $(shell pwd)/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus --web.enable-remote-write-receiver --config.file=/etc/prometheus/prometheus.yml
\ No newline at end of file
+prom1:
```

Comment:
might be good to give these targets clear names based on what they do.

Reply:
Yes, that was a quick setup for testing. @valeriiashapoval how about wrapping this all up in a script to set up both prometheuses with one command?

Reply:
thanks @machi1990  @pb82 ! working on the script now